docs/quickstart.md

```diff
@@ -63,7 +63,7 @@ python:
 
 Finally, submit your job to Racetrack:
 ```shell
-racetrack deploy sample/ http://localhost:7102 --context-local
+racetrack deploy sample/ --remote http://localhost:7102
```

Comment:
Good question. Without `--context-local` the default strategy comes in, which is doing the same in case of deploying a sample to the localhost. So it could have always been without that flag.